packages/compiler/src/core/options.ts

```diff
@@ -11,9 +11,9 @@ export interface CompilerOptions {
   outputDir?: string;
 
   /**
-   * Path to config YAML file or folder in which to search for default tspconfig.yaml file.
+   * Path to config YAML file used, this is also where the project root should be.
    */
-  config?: string;
+  configPath?: string;
```

Comment:
I believe this is a 'breaking' change, in that, if anyone has a script that calls this option, it would be broken.  Do we need to change the name of this option, rather than adding a new one, `configuration` or `configContents`.

I think deprecating the option first would make sense

Reply:
Yeah that's fair, kinda hope this would really not be used which I seriously doubt because it is confusing on why it is needed to start with. I'll keep it liek that and maybe revisit to rename to `projectRoot` as this is its purpose.